vevestaX/vevesta.py

```diff
@@ -479,21 +479,23 @@ def __getExcelSheetData(self, fileName, sheetName):
                 modelingData = pandas.read_excel(fileName, sheet_name=sheetName, index_col=[])
                 return modelingData
 
-    def commit(self, techniqueUsed, filename=None, message=None, version=None, projectId=None, attachment=None):
+    def commit(self, techniqueUsed, filename=None, message=None, version=None, projectId=None, attachmentFlag=True):
         self.dump(techniqueUsed, filename=filename, message=message, version=version, showMessage=False)
 
         # api-endpoint
         token = open("access_token.txt", "r").read()
         backend_url = 'https://api.matrixkanban.com/services-1.0-SNAPSHOT'
 
         # upload attachment
-        if attachment:
-            files = {'file': open(attachment, 'rb')}
-            headers_for_file = {'Authorization': 'Bearer '+token}
-            params = {'taskId': 0}
-            response = requests.post(url=backend_url+'/Attachments', headers=headers_for_file, params=params, files=files)
-            attachments = list()
-            attachments.append(response.json())
+        if attachmentFlag:
+            filename = self.get_filename()
```

Comment:
The problem here is that wrong file name might exist in the variable...u need to check if the file exist in the path then read it ...else mostly it could be you have wrong file name